packages/language-server/src/__test__/completion.test.ts

```diff
@@ -2772,7 +2735,7 @@ suite('Completions', function () {
             }`,
           expected: {
             isIncomplete: false,
-            items: [mapProperty],
+            items: [mapProperty, sortProperty],
```

Comment:
I might not have the context to understand why `sortProperty` has been added here

Reply:
Here it's about `@unique(|)`
https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#unique
>Note: In version 3.5.0 and later, new length and sort arguments are available as a Preview feature.